src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+
+import { useState } from 'react';
+import React from 'react';
+import Navbar from './UI Component/header';
```

Comment:
You should not have spaces in your folder naming. Use a `-` or `_` to separate words or use camel casing e.g `componentName` or Pascal casing `ComponentName`